open-api/rest-catalog-open-api.yaml

```diff
@@ -3260,6 +3260,28 @@ components:
           additionalProperties:
             type: string
 
+    FineGrainedDataProtectionRules:
+      type: object
+      description: >
+        Fine-grained data protection rules for a table as result of fine grained policy evaluation at the catalog end based on the clients access rights.
+        
+        The client SHOULD use these rules to enforce fine-grained data protection like column and row level access when reading data from the table.
```

Comment:
If we go with the above proposed name,

"Protection policy for a table including fields and row filter expressions. The client should use these rules to project the specified columns and filter the rows based on the expression"

>as result of fine grained policy evaluation at the catalog end based on the clients access rights.

IMO this feels a bit overspecified and is implicit since it's the response from a server